project/make/dynbinary_gccgo

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set +x
+
+DEST=$1
+
+echo "SOURCING dockerinit"
+source "$(dirname "$BASH_SOURCE")/.dockerinit_gccgo"
```

Comment:
Can we please remove the extra debugging output and add the same simple `DOCKER_CLIENTONLY` check that `dynbinary` has?  The diff between this file and `dynbinary` should be essentially just adding `gccgo` in a couple places.